tests/e2e/report-date-filter.js

```diff
@@ -100,25 +101,16 @@ describe('Filters reports', () => {
 
   it('by date', () => {
     commonElements.goToReports();
-    helper.waitElementToPresent(element(by.css('#reports-list .unfiltered li:first-child')));
+    helper.waitElementToPresent(reportsTab.firstReport);
 
-    let clear = '';
-    for (let i = 0; i < 20; i++) {
-      clear += protractor.Key.BACK_SPACE;
-    }
+    reportsTab.filterByDate('05/16/2016', '05/17/2016');
```

Comment:
Does that crazy date format actually work?! I thought it had to be D/M/Y...

It would be more robust to pass a Date or Moment here, so if the date format changes it only has to change in the PO and not every test that uses it.

```suggestion
    reportsTab.filterByDate(moment('2016-05-16'), moment('2016-05-17'));
```

Reply:
Apparently, this is the default `daterangepicker` format, which we never change: 
https://github.com/medic/cht-core/blob/master/webapp/src/js/services/search-filters.js#L175
https://www.daterangepicker.com/#examples

Reply:
My main motivation was about making it easier to interact with the PO by passing in a rich object rather than a particularly formatted string. The way it is now, the next person who wants to use this API will have to know exactly how to format the string to work with the `daterangepicker` format. If you decide to take a moment, then the PO would have logic to format it to `MM/DD/YYYY`, so no app changes would be required.

Reply:
Cool, I see what you mean now. I'm passing a moment object and then calling .format with the expected output.